src/components/check-your-email/index.njk

```diff
@@ -9,52 +9,52 @@
 
 {% block pageContent %}
 
-{% include "common/errors/errorSummary.njk" %}
+    {% include "common/errors/errorSummary.njk" %}
 
-<h1 class="govuk-heading-l govuk-!-margin-top-0 govuk-!-margin-bottom-3">{{'pages.checkYourEmail.header' | translate}}</h1>
+    <h1 class="govuk-heading-l govuk-!-margin-top-0 govuk-!-margin-bottom-3">{{'pages.checkYourEmail.header' | translate}}</h1>
 
-{{ govukInsetText({
+    {{ govukInsetText({
     html: 'pages.checkYourEmail.text' | translate + '<span class="govuk-body govuk-!-font-weight-bold">' + email + '</span>'
   }) }}
 
-<p class="govuk-body">{{'pages.checkYourEmail.info.paragraph1' | translate}}</p>
-<p class="govuk-body">{{'pages.checkYourEmail.info.paragraph2' | translate}}</p>
-<p class="govuk-body">{{'pages.checkYourEmail.info.paragraph3' | translate }}</p>
+    <p class="govuk-body">{{'pages.checkYourEmail.info.paragraph1' | translate}}</p>
+    <p class="govuk-body">{{'pages.checkYourEmail.info.paragraph2' | translate}}</p>
+    <p class="govuk-body">{{'pages.checkYourEmail.info.paragraph3' | translate }}</p>
 
-<form action="/check-your-email" method="post" novalidate>
+    <form action="/check-your-email" method="post" novalidate="novalidate">
 
-<input type="hidden" name="_csrf" value="{{csrfToken}}"/>
-<input type="hidden" name="email" value="{{email}}"/>
-{{ govukInput({
+        <input type="hidden" name="_csrf" value="{{csrfToken}}"/>
+        <input type="hidden" name="email" value="{{email}}"/>
+        {{ govukInput({
     label: {
         text: 'pages.checkYourEmail.code.label' | translate
     },
     classes:"govuk-input--width-5",
     id: "code",
     name: "code",
     value: code,
+    inputmode: "numeric",
+    pattern: "[0-9]*",
```

Comment:
does it help if this is limited to six chars? or would it do weird stuff?

Reply:
I'm not entirely sure but I can ask the Design System team. The pattern used here seemed to fit with what's presented in the Design System [example where there is a specified character range](https://design-system.service.gov.uk/components/text-input/#numbers).